tests/testsuite/cargo_new/add_member_to_workspace/absolute_package_path/in/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub fn add(left: usize, right: usize) -> usize {
```

Comment:
Could you pare these tests down
- Remove the README
- Remove the `lib.rs` since the workspace is virtual anyways

(ditto for others)